website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -10,6 +10,17 @@ To-do:
 - think about whether some of these should be outside of models
 --->
 
+| Configuration    | Description    | 
```

Comment:
a few comments:
- [ ] sugg having a nice intro sentence to this so users know what the table is? 
- [ ] also for consistency purposes, should you consolidate some of the rows so it matches the headers? right now, it seems like you've added multiple rows for a header (e.g. sortkey).  some of the other rows just highlight the h2s 

so for example, suggest:

Add intro sentence here.....

| Configuration    | Description    | 
|------------------|----------------|
|[Performance optimization](/reference/resource-configs/redshift-configs#performance-optimizations)| Use [sort and dist](#using-sortkey-and-distkey) in model configs to optimize Redshift performance. |
....

Reply:
also, if there are more performance opt added in the future as h3s, it'll be easier to maintain